rj_gameplay/stp/role/assignment/naive.py

```diff
@@ -0,0 +1,164 @@
+from collections import defaultdict
+from math import isfinite
+from typing import Tuple, List
+
+import scipy.optimize
+import numpy as np
+
+import stp.role as role
+import stp.role.assignment as assignment
+import stp.game_state
+
+SortedRequests = List[assignment.FlatRoleRequests]
+
+
+class NaiveRoleAssignment(assignment.IRoleAssignment):
+    @staticmethod
+    def get_sorted_requests(requests: assignment.FlatRoleRequests) -> SortedRequests:
+        """Returns a list of FlatRoleRequests sorted in ascending priority order.
+        :param requests:
+        :return:
+        """
+        role_id: assignment.RoleId
+        request: role.RoleRequest
+
+        sorted_requests: SortedRequests = [
+            {} for _ in range(role.Priority.NUM_PRIORITIES)
+        ]
+
+        for role_id, request in requests.items():
+            sorted_requests[int(request.priority)][role_id] = request
+
+        return sorted_requests
+
+    @staticmethod
+    def compute_costs_matrix(
+        free_robots: np.ndarray,
+        flat_requests: assignment.FlatRoleRequests,
+        game_state: stp.game_state.GameState,
+        prev_results: assignment.FlatRoleResults,
+    ) -> np.ndarray:
+        """Computes the m x n cost matrix corresponding to the passed in free robots and
```

Comment:
What are `m` and `n`? (I know from reading the code, but put it in the docstring)

Reply:
Done.